pages/docs/overlay/modal.mdx

```diff
@@ -446,6 +446,72 @@ there are certain situations where you might not want the modal to trap focus.
 
 To prevent focus trapping, pass `trapFocus` and set its value to `false`.
 
+### Styling the backdrop
+
+In the case you want to keep the users focus on the modal you can use the
+`backdropFilter` property. To do this you can add one or multiple filters and
+change the `bg` property, which is per default set to `blackAlpha.600`.
+
+Another approach is to use e.g. the `backdropBlur` or other similar defined
+properties. To do so you need to set the `backdropFilter` to `auto`.
```

Comment:
The backdrop's background by default is set to `blackAlpha.600`, but if you want to achieve a different style you can also use the backdrop style props, like `backdropBlur`, `backdropBrightness`, `backdropContrast`, `backdropHueRotate`, `backdropInvert`, and `backdropSaturate`. To use these style props, you'd have to set the `backdropFilter` prop to `auto`.